### PR TITLE
🔧 MAINT: Pin ipykernel to ~v5.5

### LINKED ISSUE #346: Tests failing due to regression in ipynb error messages
### Describe the problem

The tests are currently failing because the output syntax for errors seem to have changed:

https://github.com/executablebooks/MyST-NB/runs/3210843582#step:5:233

I tried debugging this locally but it seems like the `ipykernel_NNN` part of the error message changes each time the test is run, so always fails.

@chrisjsewell suggested the following may be the issue:

> The ipykernel now being installed is ipykernel-6.0.3, whereas previously it was ipykernel-5.5.5, so the easiest thing would be to just add `ipykernel~=5.5` to the `testing` extra

### PR DESCRIPTION
**Pull request by choldgraf**
closes #346

### CODE REVIEW

**Review by chrisjsewell — DISMISSED** [review]
Cheers! (Providing the tests now pass)

if you could also open an issue for eventually updating the tests to ipykernel v6 (although I think this has dropped support for python 3.6, so we would also need to drop that)

File: setup.cfg [hunk 1]
```diff
@@ -95,6 +95,7 @@ testing =
     pytest-cov~=2.8
     pytest-regressions
     sympy
+    ipykernel~=5.5
```
Comment (chrisjsewell) [inline setup.cfg:98]: one nitpick: can we move this up to be in alphabetical order 😬

**Comment by choldgraf** [comment]
thanks for the review - I opened up https://github.com/executablebooks/MyST-NB/issues/348 and moved the dep

**Comment by choldgraf** [comment]
ok gonna merge this one since we're fixing a failing test on master...

**Pull request merged**